### DISCUSSION TITLE
create-astro `--add` option

### DISCUSSION THREAD

**Discussion by ascorbic** [withastro/roadmap#903] [Proposal]
# Summary

Add a new flag to the `create-astro` tool that adds integrations to the newly-created site without needing to run `astro add` separately.

# Background & Motivation

This came up when I was documenting how to create a new Astro site on Netlify, but it applies to any integration. I would like to be able to provide users a one-line command to bootstrap a new site. Currently this is either a multi-step process (create site, then cd into the directory and run astro add), or need a custom template. This would be a more lightweight option, designed for including in docs and automated installs/tests.

# Goals

- allow a new site to be bootstrapped, complete with integrations, in a single command.

# Example

`npm create astro@latest --add tailwind`

**Comment by Princesseuh** [comment]
Sounds great, I'd love to see this!
